templates/default/netnode/netnodeevents.html

```diff
@@ -0,0 +1,39 @@
+<!--// $Id$ //-->
+<table class="lmsbox lms-ui-sortable lms-ui-tab-container" id="netnodeevents">
+    <thead>
+	<tr class="hand lmsbox-titlebar" data-lmsbox-content="netnodeeventspanel">
+		<td class="bold">
+			{icon name='timetable' class='lms-ui-sortable-handle' label='Events'}
+			({if !empty($queue)}{count($queue)}{else}0{/if})
```

Comment:
Dlaczego nie po prostu `...|size`?

Reply:
Bo jeśli `$queue` nie istnieje to `$queue|size` zwróci ostrzeżenie.